clang-plugin/MozsearchIndexer.cpp

```diff
@@ -2591,15 +2597,60 @@ class IndexConsumer : public ASTConsumer,
     }
   }
 
+  bool ArityMatchesCurrentCallExpr(const Expr *E, const NamedDecl *Candidate) {
+    const auto IsCurrentCalle = CurrentCall && E == CurrentCall->getCallee();
```

Comment:
Oh yes indeed. Fixed that.
I also renamed ArityMatchesCurrentCallExpr to arityMatchesCurrentCallExpr (camelCase instead of CamelCase). I'm not entirely clear on the LLVM style convention on function names.